client/src/reducers/contributors.js

```diff
@@ -58,7 +58,14 @@ const initialContributorsState = {
       'I also love reading sci-fi novels and watching reality TV.',
     samPic
   ),
-  4: createContributor('Teammate C Name', null, null, null),
+  4: createContributor(
+     'Anthony Edwards',
```

Comment:
It looks like the lines here might have 1 extra indentation. They should be 2 spaces from the previous line.

Reply:
I think that line had shown a deletion due to me moving my name down by one line on my second commit.